src/components/Pagination/Pagination.tsx

```diff
@@ -62,35 +83,39 @@ const Wrapper = styled.div`
   display: inline-block;
 `
 const List = styled.ul`
-  ${({ theme }: InjectedProps) => {
-    const { frame, palette } = theme
-
-    return css`
-      display: flex;
-      border-radius: ${frame.border.radius.s};
-      border: 1px solid ${palette.BORDER};
-      margin: 0;
-      padding: 0;
-
-      & > li {
-        margin: 0;
-        padding: 0;
-        list-style: none;
-
-        &:not(:first-child) {
-          border-left: ${frame.border.default};
-        }
-
-        &:first-child > .PaginationItem {
-          border-top-left-radius: ${frame.border.radius.s};
-          border-bottom-left-radius: ${frame.border.radius.s};
-        }
-
-        &:last-child > .PaginationItem {
-          border-top-right-radius: ${frame.border.radius.s};
-          border-bottom-right-radius: ${frame.border.radius.s};
-        }
+  display: flex;
+  margin: 0;
+  padding: 0;
+  > li {
+    list-style: none;
+    margin: 0 4px;
+    &.prev {
+      margin-right: 12px;
```

Comment:
[MUST] Please user `pxToRem` method to set numbers. like below.

```
margin-right: ${theme.size.pxToRem(12)}
```

Reply:
And we have some "spacing" rules.
Please check InVision carefully.
( In this case, you should set `16px` )